x-pack/plugins/apm/public/components/app/TransactionDetails/Transaction/WaterfallContainer/Waterfall/WaterfallItem.tsx

```diff
@@ -41,8 +46,13 @@ const Container = styled<IContainerStyleProps, 'div'>('div')`
   background-color: ${props =>
     props.isSelected ? theme.euiColorLightestShade : 'initial'};
   cursor: pointer;
+
   &:hover {
     background-color: ${theme.euiColorLightestShade};
+
+    ${DurationStyled} {
+      display: block;
```

Comment:
You talked about using `visibility` instead of `display` to avoid the text to "jump". Was that not needed?